packages/tests/tests/light-push/index.node.spec.ts

```diff
@@ -135,17 +140,6 @@ const runTests = (strictNodeCheck: boolean): void => {
       });
     });
 
-    it("Fails to push message with empty content topic", async function () {
```

Comment:
why did we remove this?

Reply:
because this does not belong here and must be a unit test ;) 
https://github.com/waku-org/js-waku/blob/163bea56c3e73e8f4f151f134ea7f997f2112912/packages/core/src/lib/message/version_0.spec.ts#L107